landscape/explanation/glossary/distribution.md

```diff
@@ -0,0 +1,23 @@
+# Distribution
+
+In Landscape, a distribution refers to a specific release of the Ubuntu operating System. You can reference it with either the code name (for example jammy), the version number such as 22.04.4, or the actual name (Ubuntu) depending on your current use case.
+
+Distributions are useful for many reasons in Landscape. Many Landscape APIs require you to define a distribution in order to use them. Distributions are also important if you want to perform an action such as [repository mirroring](https://ubuntu.com/landscape/docs/explanation-about-repository-mirroring).
+
+## Landscape APIs That Require a Distribution
+The following is a list of some Landscape APIs that need a distribution to function properly:
```

Comment:
```suggestion
The following is a list of example Landscape API methods that need a distribution:
```

Reply:
"Landscape APIs" would refer to different actual APIs (e.g., if we had a different API for admin actions versus customer actions, or whatever), not the different methods (or "endpoints"). I used the term "methods" for consistency here with the relevant API documentation.

We technically do have two APIs now, but it's only because we're transitioning out of our legacy API (the one in this document) to a new REST API. (Just for some extra info!)